server/routes.js

```diff
@@ -7,7 +7,8 @@ const HTML_FILE = path.join(DIST_DIR, 'index.html');
 
 export default (app) => {
 
-    app.get('/bgg/:listType/:listOption', createGamesList);
+    app.get('/bgg/random/:listType/:listOption', getRandomGame);
+    app.get('/bgg/list/:listType/:listOption', getCompatibleList);

```

Comment:
Now that we have an endpoint for getting the full list of games anyway, do we want the random game selection to be done on the backend?

I'm not suggesting the change be made now, but I think there are a number of benefits to doing it on the front-end.  It would make a lot of potential features easier.

For example, if they want to rule out specific games (that they've maybe played recently).  This could even be an option when a random game is shown to them ("Show me a different game").

It would also make caching possible, which will make everything a bit snappier.